parlai/agents/drqa/utils.py

```diff
@@ -7,6 +7,20 @@
 import time
 import unicodedata
 from collections import Counter
+import spacy
+
+NLP = spacy.load('en')
+pos_list = [
+    'DET', 'ADP', 'PART', 'ADJ', 'PUNCT', 'INTJ', 'NOUN', 'ADV', 'X', 'PRON',
+    'PROPN', 'VERB', 'CONJ', 'SPACE', 'NUM', 'SYM', 'CCONJ'
+]
+ner_list = [
+    'QUANTITY', 'PRODUCT', 'EVENT', 'FACILITY', 'NORP', 'TIME', 'LANGUAGE',
+    'ORG', 'DATE', 'CARDINAL', 'PERSON', 'ORDINAL', 'LOC', 'PERCENT', 'MONEY',
+    'WORK_OF_ART', 'GPE', 'FAC', 'LAW'
+]
+pos_dict = {i: pos_list.index(i)/len(pos_list) for i in pos_list}
+ner_dict = {i: ner_list.index(i)/len(ner_list) for i in ner_list}
```

Comment:
It doesn't make much sense to me to encode the feature as a fixed scalar between 0 and 1. My guess is the benefit you got from this is really just somewhat binary NE not NE (no typing)...